src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage handles the writing and reading of tasks to and from a specified file. It can convert the tasks
+ * in file line format to Task objects.
+ */
+public class Storage {
+    private String filePath;
+    private ArrayList<String> lines;
```

Comment:
Is it really necessary to build an array list in your storage, or is it enough to have a `.txt` file? 😄